synapse/storage/state.py

```diff
@@ -322,20 +583,14 @@ def get_state_groups(self, room_id, event_ids):
         })
 
     @defer.inlineCallbacks
-    def _get_state_groups_from_groups(self, groups, types, members=None):
+    def _get_state_groups_from_groups(self, groups, state_filter=StateFilter.all()):
```

Comment:
given that this is a private method, and a value is always passed for `state_filter`, I would be inclined to drop the default